Mathlib/RingTheory/TensorProduct.lean

```diff
@@ -213,40 +213,15 @@ variable [CommSemiring R]
 variable [NonUnitalNonAssocSemiring A] [Module R A] [SMulCommClass R A A] [IsScalarTower R A A]
 variable [NonUnitalNonAssocSemiring B] [Module R B] [SMulCommClass R B B] [IsScalarTower R B B]
 
-/-- (Implementation detail)
-The multiplication map on `A ⊗[R] B`,
-for a fixed pure tensor in the first argument,
-as an `R`-linear map.
--/
-def mulAux (a₁ : A) (b₁ : B) : A ⊗[R] B →ₗ[R] A ⊗[R] B :=
-  TensorProduct.map (LinearMap.mulLeft R a₁) (LinearMap.mulLeft R b₁)
-#align algebra.tensor_product.mul_aux Algebra.TensorProduct.mulAux
```

Comment:
These should be `noalign`ed

bors r-

bors d+

Reply:
do you mean I should keep these code but just `noalign` them? Note that I have added 
```
#noalign algebra.tensor_product.mul_aux
#noalign algebra.tensor_product.mul_aux_apply
```